packages/core/src/views/shared/NetworkSelector.svelte

```diff
@@ -134,7 +135,7 @@
         >
       {/if}
       {#each chains as chain (chain.id)}
-        <option value={chain.id}>{chain.label}</option>
+        <option value={chain.id}>{chain.label || chainIdToLabel[chain.id]}</option>
```

Comment:
If the chain isn't in the natively supported chains do we want to display something? Maybe just chainId?